src/hotspot/share/classfile/classLoaderData.cpp

```diff
@@ -599,7 +599,7 @@ void ClassLoaderData::unload() {
   free_deallocate_list_C_heap_structures();
 
   // Clean up class dependencies and tell serviceability tools
-  // these classes are unloading.  Must be called
+  // these classes are unloading.  This must be called
```

Comment:
```suggestion
  // these classes are unloading. This must be called
```

Reply:
Hotspot code style allows comments with both two spaces and one space between sentences in a paragraph. I did not want to change the style there.